sql/_05_plcsql/_01_testspec/_01_basic_structure/_01_create_procedure/_12_comment/cases/02_param_comment.sql

```diff
@@ -0,0 +1,57 @@
+as
+begin
+    dbms_output.put_line('proc2 param: ' || a);
+    test_proc1(3);
+end;
+
+-- recursive
+create or replace procedure test_proc3(num int comment 'recursive procedure')
+as
+begin
+    if num = 0 then
+        dbms_output.put_line('recursive done');
+    else
+        test_proc3(num-1);
+    end if;
+    dbms_output.put_line('param: ' || num);
+end;
+
+-- maximum test
+-- compile error: length over
+create or replace procedure test_proc4(num int comment '_0123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890')
+as
+begin
+    dbms_output.put_line('maximum comment length');
+end;
+-- maximum length: 1024 byte
+create or replace procedure test_proc4(num int comment '0123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890123456768901234567689012345676890')
+as
+begin
+    dbms_output.put_line('maximum comment length');
+end;
+
+call test_proc1();
+call test_proc2(10);
+call test_proc3(5);
+call test_proc4(99);
+
+select sp_name, comment from db_stored_procedure_args where comment is not null;
+
```

Comment:
select sp_name, arg_name,  comment from db_stored_procedure_args where comment is not null;